src/monsters.cpp

```diff
@@ -186,6 +239,175 @@ void MonsterType::createLoot(Container* corpse)
 	corpse->startDecaying();
 }
 
+void MonsterType::clone(MonsterType* root, MonsterType* child, const std::string& type) {
```

Comment:
Consider changing the argument names to source and target. It's not clear with root and child which one is which.